.github/workflows/update.yml

```diff
@@ -25,5 +45,6 @@ jobs:
         id: update
         uses: DeterminateSystems/update-flake-lock@v21
         with:
-          pr-title: "Update flake.lock"
+          pr-title: "Update flake.lock (${{ matrix.branch }})"
           token: ${{ secrets.GH_TOKEN_FOR_UPDATES }}
+          branch: "update_flake_lock_action-${{ matrix.branch }}"
```

Comment:
Does the update-flake-lock action use `github.ref` or the _actually_ checked-out HEAD ref as it's target/base branch when opening the PR?

Reply:
Upstream [claims](https://github.com/DeterminateSystems/update-flake-lock/blob/222f041780a88f7f379bd7a3c3327585e435db5b/action.yml#L17) it's the latter, but I haven't tested or looked at the implementation to be sure.

Reply:
I tested, it uses the currently checkout out ref